data/Localization/german_(germany)/global.ini

```diff
@@ -32273,7 +32273,7 @@ PU_HURSTON_MG_Comms_ReturnItems_FailBounty_IG_001_ThatsItI=Das war's. Ich habe v
 PU_HURSTON_MG_Comms_ReturnItems_FailBounty_IG_002_AfterMultipleDemands=Nach mehrfacher Aufforderung zur Rückgabe von Hurston-Eigentum sehe ich mich gezwungen, diese Angelegenheit den zuständigen Behörden zu melden.
 PU_HURSTON_MG_Comms_ReturnItems_LastWarning_IG_001_ThisIsYour=Dies ist Ihre letzte Chance. Ich meine es wirklich ernst. Ich werde das auf jeden Fall melden.
 PU_HURSTON_MG_Comms_ReturnItems_LastWarning_IG_002_IDontKnow=Ich weiß nicht, was Sie glauben, was Sie da tun, aber das ist nicht Ihr Eigentum. Geben Sie es zurück.
-PU_HURSTON_MG_Comms_ReturnItems_Reminder_IG_001_PropertyThatBelongs=Eigentum, das Hurston Dynamics gehört, wurde Ihnen für die Zwecke dieses speziellen Vertrags überlassen. Sie müssen es unverzüglich zurückgeben.
+PU_HURSTON_MG_Comms_ReturnItems_Reminder_IG_001_PropertyThatBelongs=Eigentum, das Hurston Dynamics gehört, wurde Ihnen für die Zwecke dieses speziellen Auftrags überlassen. Sie müssen es unverzüglich zurückgeben.
 PU_HURSTON_MG_Comms_ReturnItems_Reminder_IG_002_YouAreIn=Sie sind im Besitz von Hurston Dynamic-Eigentum. Geben Sie es zurück. Jetzt.
 PU_HURSTON_MG_ConvoRejoin_IG_001_GuessWeCan=Ich schätze, wir können das noch einmal versuchen...
 PU_HURSTON_MG_ConvoRejoin_IG_002_OhICan=Oh, ich kann fertig werden? Das ist toll, danke. Wie auch immer...
```

Comment:
Eine Änderung in "Oh, ich kann es zu Ende bringen? Das ist toll, danke. Jedenfalls..." wäre vielleicht angebracht